terraform/sso-admin-account-assignments.tf

```diff
@@ -140,12 +140,44 @@ locals {
         aws_organizations_account.hmpps-community-rehabilitation-unpaid-work-production
       ]
     },
-    # MOJ Official Technical Operations (Networking)
+    # MOJ Official Technical Operations (Networking) and Cloud Ops team
     {
       github_team    = "moj-official-techops"
       permission_set = aws_ssoadmin_permission_set.administrator-access
       accounts = [
-        aws_organizations_account.moj-official-network-operations-centre
+        aws_organizations_account.moj-official-network-operations-centre,
+        aws_organizations_account.wptpoc,
+        aws_organizations_account, workplace-tech-proof-of-concept-development
```

Comment:
Slight formatting issue here - should be `aws_organizations_account.workplace-tech-proof-of-concept-development`

Reply:
fixed